package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "skeletor-plugin-patternlab",
```

Comment:
Can we change this to include the new scoped package naming conventions. so `@skeletor/plugin-patternlab`